FuelSDK/client.py

```diff
@@ -50,8 +53,13 @@ def __init__(self, get_server_wsdl = False, debug = False, params = None, tokenR
         else:
             logging.getLogger('suds').setLevel(logging.INFO)
 
+        self.set_class_variables(get_server_wsdl, params, tokenResponse)
+
+    def set_class_variables(self, get_server_wsdl, params, tokenResponse):
```

Comment:
Can this be renamed to *configure_client*?

Reply:
Done